blockem/blockem.php

```diff
@@ -137,20 +139,25 @@ function blockem_prepare_body_content_filter(\Friendica\App $a, &$hook_data)
 	}
 }
 
-function blockem_display_item(&$a,&$b) {
-	if(strstr($b['output']['body'],'id="blockem-wrap-'))
+function blockem_display_item(App $a, array &$b = null)
```

Comment:
Can you please remove the `= null`? If there's no value provided, this function should error out directly.

Reply:
Ops ...

Reply:
No, I'm sorry. I found out that I badly need it for my installation to work.

Reply:
What is the link supposed to go to?

Reply:
I found an error when I left out ` = null` and I put it into commit message.

Here it is again:
````
- Fatal error: Uncaught TypeError: Argument 2 passed to blockem_display_item()
  must be of the type array, null given, called in /var/www/../htdocs/src/Core/Addon.php
  on line 246 and defined in /var/www/../htdocs/addon/blockem/blockem.php on
  line 142
````

Reply:
This is caused in `include/conversation.php` line 724. I don't know why there is a wrong `Addon::callHooks` call here, but we should fix it over there instead of suppressing the error message in the addon when adding a correct type hint.

It's good actually, type hints are allowing another level of debug.

Reply:
I included this in my current PR where I work on all the warnings.